server/src/main/java/org/elasticsearch/synonyms/SynonymsAPI.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.synonyms;
+
+import org.elasticsearch.common.util.FeatureFlag;
+
+public class SynonymsAPI {
```

Comment:
I wonder if this class needed only to store `SYNONYMS_API_FEATURE_FLAG`?

 If yes, why can't we keep `SYNONYMS_API_FEATURE_FLAG` in `SynonymsManagementAPIService`?

Reply:
Answered above - seemed cleaner to export just that FF as part of a class, instead of using the whole `SynonymsManagementAPIService`, that exports more operations.